.travis-ci.sh

```diff
@@ -105,16 +107,16 @@ on the github pull request.
 ------------------------------------------------------------------------
 EOF
   # check that Changes has been modified
-  git diff $TRAVIS_MERGE_BASE..$TRAVIS_PR_HEAD --name-only --exit-code Changes \
+  git diff "$TRAVIS_MERGE_BASE".."$TRAVIS_PR_HEAD" --name-only --exit-code Changes \
```

Comment:
Please pick one style and stick to it. My preference goes to the style used in line 145 of this file:
`"$TRAVIS_MERGE_BASE..$TRAVIS_PR_HEAD"`

Reply:
Done.